codegen/debug.go

```diff
@@ -119,9 +125,12 @@ func NewDebugger(c *client.Client, w io.Writer, unbufferedReader io.Reader) Debu
 
 				command = strings.Replace(command, "\n", "", -1)
 
-				if command == "" {
+				if command == "" && previousCommand == "" {
 					continue
+				} else if command == "" {
+					command = previousCommand
 				}
+				previousCommand = command
```

Comment:
This allows us to just hit enter to repeat the last command, it is frustrating typing `next\n` over and over to step through the program.  Most debuggers allow you to type `next\n`, then subsequent `\n` will repeat that command.

Reply:
Should we limit it to commands where this makes sense... for example I don't think we'd want to repeat `break` or `exec`?

Reply:
Not sure, I think "always execute previous command" is easier to reason about than special-casing a few commands.  I believe this is consistent behavior with gdb and other debuggers I have used, although I am not 100% positive on that.

Reply:
I remember hating this gdb behavior, but I won't object.

Reply:
I don't object to having special cases eventually, I think we just want to live with it for a bit first so we can more clearly identify what frustrates us.  Now that the debugger is "useful" with the exec functionality I think it will get more use soon.